### PR TITLE
config answer fields

### PR DESCRIPTION
**Pull request by fiorda**

### CODE REVIEW

File: pybossa/view/projects.py [hunk 1]
```diff
@@ -2925,3 +2925,34 @@ def assign_users(short_name):
...
+    """Returns Project Stats"""
+    project, owner, ps = project_by_shortname(short_name)
+    pro = pro_features()
+    ensure_authorized_to('update', project)
+
+    if request.method == 'POST':
+        try:
+            project.info['answer_fields'] = json.loads(request.data)
+            project_repo.save(project)
+            flash(gettext('Configuration updated successfully'), 'success')
```
Comment (dchhabda) [inline pybossa/view/projects.py]: as its config changes, it can be audited.

File: pybossa/view/projects.py [hunk 2]
```diff
@@ -2925,3 +2925,34 @@ def assign_users(short_name):
...
+        except Exception:
+            flash(gettext('An error occurred.'), 'error')
+
+    project_sanitized, owner_sanitized = sanitize_project_owner(
+        project, owner, current_user, ps)
+    gold_fields = project.info.get('answer_fields', {})
```
Comment (dchhabda) [inline pybossa/view/projects.py]: to be consistent, can this be named as answer_fields?

File: pybossa/view/projects.py (hunk 1 above)
Comment (fiorda) [reply pybossa/view/projects.py]: yes, will add audit log

File: pybossa/view/projects.py (hunk 2 above)
Comment (fiorda) [reply pybossa/view/projects.py]: will fix

**Pull request merged**